list-licenses.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 
+from __future__ import print_function
```

Comment:
Import not used?

Reply:
This overrides the `print` function so we can use the python 3 version. I used this so we can print warning messages to stderr.

Wasn't exactly sure if this was targeting python 2 or 3 but the above should work for either version. On my side I was testing using python 2.

Reply:
I learned a thing, nice 💯